### PR TITLE
Refactor/about page

### PR DESCRIPTION
**Pull request by t-pyrope**
From Aleš:

> re: About screen and versioning.
> I revisited this topic and I came to following conclusion:
> - product name, web site link and email are generic and they should be read from configuration (as is implemented today)
>- version and build date (not release date!) is specific for a component

> This means that they cannot be part of the config.
> A version and a build date need to be wrapped into a dedicated container (eg. card) with title “User interface” (there will be more containers for other components in the future).
> The card is optional to display, the information to be read from the file that is produced by Dima build script - the build date needs to be added there, on the second line.

screencapture-localhost-3000-2021-07-27-15_51_11

screencapture-localhost-3000-2021-07-27-15_51_22

screencapture-localhost-3000-2021-07-27-15_51_36

screencapture-localhost-3000-2021-07-27-15_51_47

### CODE REVIEW

File: doc/user-interface.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Version    | 8992ba2+dirty
+```
+
+### Setup
+
+Load the page by adding a route in `asab-webui/src/modules/auth/index.js`
+
+```javascript
+app.Router.addRoute({
+	path: '/auth/interface',
```
Comment (ateska) [inline doc/user-interface.md:21]: The "User interface" card should be part of the About screen: a card below the existing card.
Not the dedicated router entry.

**Review by ateska — CHANGES_REQUESTED** [review]
The translation:
"Datum sestaveni"

... and this should be below a version

**Comment by ateska** [comment]
about

... the idea is that various components can add their cards on the about page.

**Comment by titkov49** [comment]
@t-pyrope Code looks good to me. It can be merged once all change requests above is solved. 
And can you also provide screenshots please?

**Review by titkov49 — COMMENTED** [review]
@t-pyrope Code looks good to me. It can be merged once all change requests above is solved.
And can you also provide screenshots please?

**Comment by t-pyrope** [comment]
screencapture-localhost-3000-2021-07-28-12_34_37

screencapture-localhost-3000-2021-07-28-12_33_51

yarn build

**Comment by t-pyrope** [comment]
screencapture-localhost-3000-2021-07-28-12_43_12

screencapture-localhost-3000-2021-07-28-12_42_17

**Comment by t-pyrope** [comment]
screencapture-localhost-3000-2021-07-28-13_39_09

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@t-pyrope Also, the translation of `User interface` is `Uživatelské rozhraní` and I want to know, why there is the grayish gap in the sidebar? Also dont implement `CardFooter`.

127309716-5a14b814-2ba4-450d-994d-d81f15069e6d

File: src/modules/about/UserInterfaceCard.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+						<h5>{t('UserInterfaceCard|Version')}</h5>
+					</Col>
+					<Col>
+						{version}
+					</Col>
+				</Row>
+				<hr/>
+				<Row>
+					<Col>
+						<h5>{t('UserInterfaceCard|Last build')}</h5>
```
Comment (ateska) [inline src/modules/about/UserInterfaceCard.js:50]: It is not a last build but "Build date"

**Review by ateska — CHANGES_REQUESTED** [review]
I see on screenshot that there is a "card footer" - we don't need that.

File: src/modules/about/AboutCard.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+				<h1>{title}</h1>
+			</CardHeader>
+
+			<CardBody>
+				<Row>
+					<Col>
+						<h5>{t('AboutCard|Web site')}</h5>
+					</Col>
+					<Col>
+						<Link to={webSiteLink}>{webSiteName}</Link>
```
[the remark this answers is not in the document]
Comment (t-pyrope) [reply src/modules/about/AboutCard.js:48]: Sorry, it's my Chrome extension for screenshots. It's not a grayish gap in original

Animation5

**Comment by t-pyrope** [comment]
logman

logman2

Animation5

**Review by titkov49 — APPROVED** [review]
Good job.  It can be merged.

File: src/modules/about/AboutCard.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+				<h1>{title}</h1>
+			</CardHeader>
+
+			<CardBody>
+				<Row>
+					<Col>
+						<h5>{t('AboutCard|Web site')}</h5>
+					</Col>
+					<Col>
+						<a href="https://teskalabs.com" target="_blank" rel="noopener noreferrer">
```
Comment (Pe5h4) [inline src/modules/about/AboutCard.js:45]: @t-pyrope I dont think the link nor the text should be `hardcoded` here. It should go to the config as well.

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

File: src/modules/about/AboutCard.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+							teskalabs.com
+						</a>
+					</Col>
+				</Row>
+				<hr/>
+				<Row>
+					<Col>
+						<h5>{t('AboutCard|Vendor')}</h5>
```
Comment (ateska) [inline src/modules/about/AboutCard.js:53]: Vendor should go first, then the website and then email.

**Review by ateska — CHANGES_REQUESTED** [review]
Last remark, then mergeable from my perspective.

**Comment by t-pyrope** [comment]
logman

logman-cz

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**